sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -580,12 +589,19 @@
     .setDeleteHistoricalData(true)
     .create();
 
+  /**
+   * 
+   * @deprecated since 5.2 soon to be removed
+   */
+  @Deprecated
   public static final String COVERAGE_LINE_HITS_DATA_KEY = "coverage_line_hits_data";
 
   /**
    * Key-value pairs, where key - is a number of line, and value - is a number of hits for this line.
    * Use {@link CoverageMeasuresBuilder} to build measure for this metric.
+   * @deprecated since 5.2 soon to be removed
    */
+  @Deprecated
   public static final Metric<String> COVERAGE_LINE_HITS_DATA = new Metric.Builder(COVERAGE_LINE_HITS_DATA_KEY, "Coverage hits by line", Metric.ValueType.DATA)
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)